src/main/java/org/veupathdb/service/eda/access/service/userdataset/UserDatasetIsaStudies.java

```diff
@@ -15,6 +16,8 @@ public class UserDatasetIsaStudies {
 
   // pseudo column name returned by SQL
   private static final String IS_OWNER_COL = "is_owner";
+  private static final String VDI_CONTROL_SCHEMA_TEMPLATE_STRING = "\\$VDI_CONTROL_SCHEMA\\$";
```

Comment:
Do you need these if using %s for macro substitutions/String format?  What are these constants used for?

Reply:
These are for the directory structure in the resources directory. Most queries have the schema in their path structure but these ones can't since the schema is dynamic.